src/components/buoy-telemetry/BuoyMap.vue

```diff
@@ -35,42 +114,121 @@ const store = inject("store") as BuoyStore;
 const props = defineProps<{
   samples: Sample[];
   formattedDate: string;
-  variable: string;
+  showChlorophyll?: boolean;
+  showWind?: boolean;
 }>();
 
 const config = {
   chlorophyll: {
-    varDomain: [0, 45],
-    markerSize: [0.25, 0.65],
-    markerColors: ["#93C572", "#32CD32", "#355E3B"], // pistachio, lime green, hunter green
+    varDomain: [0, 10],
+    markerSize: [0.2, 0.75],
+    markerColors: ["#32CD32", "#355E3B"], // lime green, hunter green
   },
 };
 
-const annotatedSamples = computed(() => {
-  const { varDomain, markerSize, markerColors } = config.chlorophyll;
-  const domain = varDomain;
-  const sqrtScale = scaleSqrt().domain(domain).range(markerSize);
-  const colorScale = scaleLinear()
-    .domain(domain)
-    .range(markerColors)
-    .clamp(true);
-  return props.samples
-    .filter((row) => row.variable === props.variable)
-    .map((row) => {
-      return {
-        ...row,
-        color: colorScale(row.value),
-        size: sqrtScale(row.value),
-      };
-    });
+const knotsPerMS = 1.94384;
+const windSpeedBins = [
```

Comment:
Maybe rename this variable so we know that this is in knots instead of just the comment?

Reply:
changed this to `windSpeedBinsKnots`